app/boards/arm/glove80/Kconfig.defconfig

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) 2021 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+if BOARD_GLOVE80_LH
+
+config BOARD
+    default "glove80 lh"
+
+config ZMK_SPLIT_BLE_ROLE_CENTRAL
+    default y
+
+config ZMK_KEYBOARD_NAME
+    default "Glove80"
```

Comment:
This gets overridden in the left/right _defconfig files.. Why bother here?

Reply:
Fair, I'll remove it.